hbw/app/javascript/packs/hbw/components/helpers/validations.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+
+export default WrappedComponent => class WithValidations extends Component {
```

Comment:
Let's make this component functional ;)
https://reactjs.org/docs/components-and-props.html#function-and-class-components